### DISCUSSION TITLE
Stepper doesn't highlight activated step

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#622] [PrimeVue] [answered]
Stepper doesn't highlight activated step but css style works as it's designed in the documentation.

**Comment by Uzeyir9695** [comment]
Also it's not possible to get activated step index with this prop: :activeStep="step"

**Comment by tugcekucukoglu** [comment, marked as answer]
`activeStep` waits for two-way data binding: `v-model:activeStep="active"`. `activeStep` property and other enhancements were added 3.40.0. You could use it with the latest version.

**Reply by Uzeyir9695** [reply]
still not working, stepper-item doesn't get activated color.

**Reply by tugcekucukoglu** [reply]
Could you send a reproducer link?

**Reply by Uzeyir9695** [reply]
Now it works. Thanks a lot.

**Reply by Uzeyir9695** [reply]
And one more question... Is it possible to validate step? For example if in the first step there is input field left without filling it out, I want to set red background for this particular step. Like Vuetify supports this kinda validation for stepper
